bugwarrior/services/__init__.py

```diff
@@ -566,11 +564,9 @@ def aggregate_issues(conf, main_section, debug):
             completion_type, args = issue
             if completion_type == SERVICE_FINISHED_ERROR:
                 target, e = args
-                log.info("Terminating workers")
-                for process in processes:
-                    process.terminate()
-                raise RuntimeError(
-                    "critical error in target '{}'".format(target))
+                log.exception(f"Aborted {target} due to critical error.")
+                currently_running -= 1
+                yield ('SERVICE FAILED', target)
             currently_running -= 1
             continue
```

Comment:
```suggestion
                log.exception(f"Aborted {target} due to critical error.")
                yield ('SERVICE FAILED', target)
            currently_running -= 1
            continue
```
The `currently_running -= 1`  will also be reached once yield is called, as the generator simply continues directly after the `yield` statement (as far as I understood generators).

Adding this changes makes the whole PR work as expected for me.